src/content/blog/community-awards-22.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Astro Community Awards 2022'
+description: 'The Astro Community Awards for 2022 have been announced, Nominated by the community to reward those Astronauts who go above and beyond the past year, read to find out who received a share of the $8500 Open Collective funding'
+publishDate: 'January 27, 2023'
+authors:
+    - fuzzy
+socialImage: '/assets/blog/astro-2/social.jpg'
```

Comment:
note to fuzzy: needs new social and cover images before merging

Reply:
on it 👊